infretis/setup.py

```diff
@@ -136,6 +136,12 @@ def write_header(config):
     size = config["current"]["size"]
     data_dir = config["output"]["data_dir"]
     data_file = os.path.join(data_dir, "infretis_data.txt")
+    if os.path.isfile(data_file):
+        for i in range(1, 1000):
```

Comment:
Why range(1, 1000)?

Reply:
Basically, if for some reason one is to start a new simulation in a folder where a simulation already have ran, we now instead of overwriting infretis_data.txt, we store the data in f"./infretis_data_{i}.txt" instead, given that f"./infretis_data_{i}.txt" does not exist. If it does, then we increment by a number.

I don't know what the upper limit should be, so i just sat an unproabable number of 1000, as I think hitting the end of a range is better than i.e. an endless while loop. Do you have any alternative ways to implement this?